admin-ui/plugins/schema/utils/validation.ts

```
@@ -0,0 +1,193 @@
+    minLength: Yup.number()
+      .nullable()
+      .transform((value, originalValue) => {
+        if (originalValue === '' || originalValue === null) {
+          return null
+        }
+        return value
+      })
+      .positive(t('errors.positive_number') || 'Must be a positive number')
+      .when('maxLength', {
+        is: (maxLength: number | null) => maxLength !== null,
+        then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)